k256/src/arithmetic/scalar.rs

```diff
@@ -242,10 +242,7 @@ impl Scalar {
     /// Returns self + rhs mod n.
     // TODO(tarcieri): use `UInt::add_mod`
```

Comment:
I was a few seconds too slow from Zulip to point out this TODO has also been addressed.

Reply:
Aah yeah, got it on `p256` but not `k256`. Thanks for pointing it out.

Reply:
Removed in 8ddcc1c